lib/ex_clubhouse/api/v2/members.ex

```diff
@@ -0,0 +1,19 @@
+defmodule ExClubhouse.API.V2.Members do
+  alias ExClubhouse.API.V2.Client
+
+  def members do
+    Client.get!("/members")
+  end
+
+  def member(id) do
+    Client.get!("/members/" <> id)
+  end
```

Comment:
It's safer to interpolate the id in to the string with `"/members/#{id}" because the runtime will implicitly call [to_string/1](https://hexdocs.pm/elixir/Kernel.html#to_string/1) and therefore can handle a few different data types that might be dropped in here. 

[Kernel.<>/2](https://hexdocs.pm/elixir/Kernel.html#%3C%3E/2) only works on binaries (the underlying basic datatype of a string) and will raise if you pass it an integer or charlist ('string' vs "string")